docs/app/client/icons/search/IconSearch.tsx

```diff
@@ -113,7 +113,9 @@ const IconSearch = () => {
       >
         <Snackbar.Message className="flex gap-2">
           <Icons.GenericCheckAlternative className="text-moon-24 text-roshi" />
-          Icon {renderIcon(lastClickedIcon)} copied for import
+          Icon {renderIcon(
```

Comment:
Please, roll back the changes. It's not relevant to the ticket

Reply:
Agreed. Done.